### PR TITLE
Handle error messages when creating and modifying elements

### PR DESCRIPTION
**Pull request by SlimaneAmar**

### CODE REVIEW

File: src/utils/rest-api.js [hunk 1]
```diff
@@ -289,7 +289,11 @@ export function fetchElementsInfos(ids) {
     }).then((response) =>
         response.ok
             ? response.json()
-            : response.text().then((text) => Promise.reject(text))
+            : response
+                  .text()
+                  .then((text) =>
+                      Promise.reject(text ? text : response.statusText)
```
Comment (JacquesBorsenberger) [inline src/utils/rest-api.js]: can't be factorized ?  like function handleJSonResponse( response ) { 
if (response.ok) return response.json()
return response.text().then((text) => Promise.reject(text ? text : response.statusText))
}

Comment (SlimaneAmar) [reply src/utils/rest-api.js]: Yes an US was created for that

**Review by JacquesBorsenberger — CHANGES_REQUESTED** [review]
factorize a little

File: src/utils/rest-api.js (hunk 1 above)
Comment (SlimaneAmar) [reply src/utils/rest-api.js]: Done

**Review by JacquesBorsenberger — APPROVED** [review]

**Review by AbdelHedhili — APPROVED** [review]

**Pull request merged**